json-rpc/docs/type_preburn_queue.md

```diff
@@ -2,7 +2,15 @@
 
 ### Attributes
 
-| Name       | Type                             | Description                                                  |
-| ---------- | -------------------------------- | -------------------------------------------------------      |
-| currency   | string                           | The currency that all preburn requests in this queue are in. |
-| preburns   | List<[Amount](type_amount.md)>   | A list of preburn requests. All in the same currency.        |
+| Name       | Type                                                   | Description                                                  |
+| ---------- | --------------------------------                       | -------------------------------------------------------      |
+| currency   | string                                                 | The currency that all preburn requests in this queue are in. |
+| preburns   | List<[PreburnWithMetadata](#type-preburnwithmetadata)> | A list of preburn requests. All in the same currency.        |
+
+
+## PreburnWithMetadata - type
```

Comment:
for above "#type-preburnwithmetadata" to work, you need change this header to "Type PreburnWithMetadata"